sources/packages/backend/apps/api/src/main.ts

```diff
@@ -24,6 +25,24 @@ async function bootstrap() {
   // Setting global prefix
   app.setGlobalPrefix("api");
 
+  // Using helmet
+  app.use(
+    helmet({
+      strictTransportSecurity: {
```

Comment:
Is the default configuration not enough in this case for `Strict-Transport-Security`.
If this is following the Wava recommendation I would say that either one would work and Wava was probably complaining about the missing configuration.

Reply:
yes, I changed the configuration and let the helmet default configuration as it should be enough.